questionnaire-zh-zh.json

```diff
@@ -0,0 +1,434 @@
+{
+  "specVersion": "2.1.1",
+  "language": "zh-zh",

```

Comment:
Per email conversation, if you could change this to zh-Hans

Reply:
done locally, sending PR later.

Reply:
Done. Sending PR later.